public/electron.js

```diff
@@ -0,0 +1,75 @@
+const path = require("path");
+
+const { app, BrowserWindow } = require("electron");
+const isDev = require("electron-is-dev");
+
+
+// Conditionally include the dev tools installer to load React Dev Tools
+let installExtension, REACT_DEVELOPER_TOOLS;
+
+if (isDev) {
+  const devTools = require("electron-devtools-installer");
+  installExtension = devTools.default;
+  REACT_DEVELOPER_TOOLS = devTools.REACT_DEVELOPER_TOOLS;
+}
+
+if (require("electron-squirrel-startup")) {
+  app.quit();
+}
+
+function createWindow() {
+  // Create the browser window.
+  const win = new BrowserWindow({
+    width: 800,
+    height: 600,
+    webPreferences: {
+      nodeIntegration: true
+    }
+  });
+
+  // and load the index.html of the app.
+  // win.loadFile("index.html");
+  win.loadURL(
+    isDev
+      ? "http://localhost:3000"
+      : `file://${path.join(__dirname, "../build/index.html")}`
+  );
+
+  // Open the DevTools.
+  if (isDev) {
+    win.webContents.openDevTools({ mode: "detach" });
+  }
+}
+
+// This method will be called when Electron has finished
+// initialization and is ready to create browser windows.
+// Some APIs can only be used after this event occurs.
+app.whenReady().then(() => {
+  createWindow();
+
+  if (isDev) {
+    installExtension(REACT_DEVELOPER_TOOLS)
+      .then(name => console.log(`Added Extension:  ${name}`))
+      .catch(error => console.log(`An error occurred: , ${error}`));
+  }
+});
```

Comment:
it is fine (though see my comment regarding to `installExtension` part), but within our repositories we use some conventions in order to the implementation seem unified.
it should be moved into an async function where please use `await` and `try ... catch`.